scheduling-manager/src/main/java/com/amazonaws/blox/scheduling/scheduler/engine/StartTask.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.scheduling.scheduler.engine;
+
+import java.util.concurrent.CompletableFuture;
+import lombok.Value;
+import software.amazon.awssdk.services.ecs.ECSAsyncClient;
+import software.amazon.awssdk.services.ecs.model.StartTaskRequest;
+import software.amazon.awssdk.services.ecs.model.StartTaskResponse;
+
+@Value
+public class StartTask implements SchedulingAction {
+  private final String clusterArn;
+  private final String containerInstanceArn;
+  private final String taskDefinition;
```

Comment:
if everything's named arn why not this one?

Reply:
Good catch, will fix